.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -72,15 +72,16 @@ body:
       description: In which operating system you are running the desktop client.
       options:
         - "Windows"
-        - "Mac OS"
+        - "macOS"
         - "Linux"
+        - "Other" (e.g. FreeBSD)
```

Comment:
Crap I messed up the quotes on this line. Fix pending in a follow-up. Currently breaks bug reports. :)